webstorage/idlharness.html

```diff
@@ -7,7 +7,7 @@
 <link rel="help" href="http://www.w3.org/TR/webstorage/#storage"/>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
-<script src="/resources/WebIDLParser.js"></script>
+<script src="/resources/webidl2/lib/webidl2.js"></script>
```

Comment:
there is a redirect in wptserve so we shouldn't have to touch anything the idlharness tests. my mistake.